File: swagger/models/custom_payment_method.py
from .base_model import BaseModel


class CustomPaymentMethod(BaseModel):
	"""
	no description available

	Attributes:

		id
		The ID of the custom payment method.

		name
		The name of the custom payment method.
	"""

	input_map = {
		'Id': 'id',
		'Name': 'name',
		}

	output_map = {
		'id': 'Id',
		'name': 'Name',
		}

	id = None
	name = None

File: swagger/Models/CustomPaymentMethod.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class CustomPaymentMethod extends BaseModel
{
	/**
	 * The ID of the custom payment method.
	 * 
	 * @var integer format:int32
	 */
	public $id;
	/**
	 * The name of the custom payment method.
	 * 
	 * @var string 
	 */
	public $name;

	protected function getInputMap()
	{
		return [
			'Id' => ['id', 'integer', 'int32'],
			'Name' => ['name', 'string', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'id' => ['Id', 'integer', 'int32'],
			'name' => ['Name', 'string', null],
			];
	}
}
